Makefile

```diff
@@ -3,6 +3,9 @@ format:
 
 check: format
 	bash ./scripts/check.sh
-
+	
 test: check
 	bash ./scripts/test.sh
+	
+db_prepare:
```

Comment:
we should have db_start, db_stop and db_prepare targets... and both mysql and mongo should be run inside the container with ports exposed.